src/main/java/core/basesyntax/ArrayList.java

```diff
@@ -1,49 +1,121 @@
 package core.basesyntax;
 
+import java.util.NoSuchElementException;
+
 public class ArrayList<T> implements List<T> {
+    private static final int DEFAULT_CAPACITY = 10;
+    private static final int ZERO_INDEX = 0;
+    private static final int USUALLY_SIZE = 1;
+    private int capacity = DEFAULT_CAPACITY;
```

Comment:
we don't need this `capacity` field - we are able to call `elementData.length` when we need